spec/labor/random_gif_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe RandomGif, type: :labor do
```

Comment:
I've never seen `type: :labor` used before. When is it appropriate to use this type of spec? 🤔

Reply:
We have a habit of tagging specs based on what they are used for. In this case the class is under the labor directory so we affix this tag. Currently, this tag is not used for anything special(that I could find) in our spec helpers. However, tags like view, system, and request allow us to include helpers like these.  
```
  config.include Devise::Test::ControllerHelpers, type: :view
  config.include Devise::Test::IntegrationHelpers, type: :system
  config.include Devise::Test::IntegrationHelpers, type: :request
```

Reply:
It's a custom name we pass so that in the coverage output and in profiling they are grouped by type. Kind of a tag in a way.

It comes from https://github.com/thepracticaldev/dev.to/pull/4864

Reply:
That's a good habit to have! 😄

That makes sense. Thank you both for the explanation and thank you @rhymes for adjusting all of those tags back in November!

Reply:
TIL!!!! Thanks @rhymes